pgx/src/dune

```diff
@@ -10,7 +10,6 @@ let () = Jbuild_plugin.V1.send @@ {|
 
 (library
  (public_name pgx)
- (wrapped false)
```

Comment:
we should probably remove the `wrapped` from pgx_async as well. I think we currentl expose `Pgx_async_test` in the public api but that's really only needed for tests. We could perhaps use core_kernel's module that indicates something is supposed to be for test only?

Reply:
Yeah I was considering just dropping Pgx_async_test entirely or moving it somewhere else. It doesn't belong in the public API at all.

Reply:
See #104